src/components/ErrorFilter.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useEffect, useState } from "react";
+import { sites } from "../siteContent";
+
+interface Status {
+  status: string;
+  value: number;
+}
+
+const ErrorFilter = () => {
+  const [allStatus, setAllStatus] = useState();
+
+  useEffect(() => {
+    const getAllSitesStatus = async () => {
+      const urlList = Object.entries(sites).map(([, pageInfo]) =>
+        pageInfo.map((p) => p.url)
+      );
+      // Collect urls into one Array
+      const urls = ([] as any).concat.apply([], urlList);
```

Comment:
hmm, why do we need to make a separate array from the `urlList` if the urlList is already an array?